task2/src/components/AppErrorBoundary/AppErrorBoundary.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class AppErrorBoundary extends React.Component {
+    constructor(props) {
```

Comment:
u use 17 react and it's possible to get rid of it at all

Reply:
> u use 17 react and it's possible to get rid of it at all

I don't know why, but babel shows the error. I think I need to add some additional settings